src/applications/disability-benefits/view-payments/components/view-payments-lists/helpers.js

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
-import moment from 'moment';
+import { format, isValid, parse } from 'date-fns';
+
+const parseFormatStr = 'MM/dd/yyyy';
+
+export const isValidDate = dateString => {
+  const parsedDate = parse(dateString, parseFormatStr, new Date());
+
+  return isValid(parsedDate);
+};
+
+export const formatDate = dateString => {
+  const formatString = 'MMMM d, yyyy';
```

Comment:
should this be ''MMMM dd, yyyy' ?

Reply:
I see references of both but more of the all caps version...not sure if it matters though?

Reply:
No, if you capitalize the d and y they both have different meanings. [Docs](https://date-fns.org/v3.6.0/docs/format)